pkg/chartutil/create.go

```diff
@@ -216,10 +216,13 @@ const defaultIgnore = `# Patterns to ignore when building packages.
 const defaultIngress = `{{- if .Values.ingress.enabled -}}
 {{- $fullName := include "<CHARTNAME>.fullname" . -}}
 {{- $svcPort := .Values.service.port -}}
-{{- if semverCompare ">=1.14-0" .Capabilities.KubeVersion.GitVersion -}}
+{{- $capabilities := .Capabilities -}}
```

Comment:
why is this necessary? I don't see a reason why we need to assign it to a variable.